cmd/goverter/main.go

```diff
@@ -14,6 +14,7 @@ func main() {
 	output := flag.String("output", "./generated/generated.go", "")
 	extends := flag.String("extends", "", "comma separated list of local or package extends")
 	packagePath := flag.String("packagePath", "", "optional full package path for the generated code")
+	commentOnStruct := flag.String("commentOnStruct", "", "optional comment on the generated struct")
```

Comment:
I rather not have this as a cli flag, it should be a comment that can be added to the converter interface. Something like  `goverter:structComment` and it should be possible to apply this multiple times like
```go
// goverter:converter
// goverter:structComment MyConverterImpl
// goverter:structComment
// goverter:structComment More detailed description
// goverter:structComment @autowire(set=converter)
type Converter interface {
	Convert(source []Input) []Output
}
```
Furthermore this feature requires some tests scenarios defined in the scenario directory and docs in docs/conversion/misc.md